python/pandemic_simulator/environment/pandemic_env.py

```diff
@@ -129,7 +159,12 @@ def step(self, action: int) -> Tuple[PandemicObservation, float, bool, Dict]:
 
         # execute the action if different from the current stage
         if action != self._last_observation.stage[-1, 0, 0]:  # stage has a TNC layout
-            regulation = self._stage_to_regulation[action]
+            
+            regulation = self._stage_to_regulation[max(0,min(4,self._last_observation.stage[-1, 0, 0]+action-1))]
```

Comment:
Also in this case, I'm not convinced this should be default?

Reply:
This was how it was mentioned in the appendix of the paper action=[-1,0,1]. This may allow faster training due to lesser actions, but I agree the previous may be better, maybe we can give a parameter to the environment constructor to allow choosing one of these options.

Reply:
Following from the comment above - just using an action wrapper to convert from [-1, 0, 1] action space to n_discrete level action space would be the way to go, while keeping this part of the code general. We used something like this in our experiments too.